backend/items/api.py

```
@@ -86,16 +89,32 @@
 
+# Initialize ES Agent Stream Rate Limiter (5 requests per day)
+es_agent_stream_rate_limiter = None
+try:
+    if redis:  # Reuse the Redis instance
+        es_agent_stream_rate_limiter = Ratelimit(
+            redis=redis,
+            limiter=FixedWindow(max_requests=5, window=86400),  # 5 per day (24 hours)
+            prefix="es_agent_stream_limit",
+        )
+        log.info("ES Agent stream rate limiter initialized successfully")
+except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)